media-video/qmplay2/qmplay2-19.12.19.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg-utils
+
+DESCRIPTION="A Qt-based video player, which can play most formats and codecs"
+HOMEPAGE="https://github.com/zaps166/QMPlay2"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/zaps166/QMPlay2"
+else
+	SRC_URI="https://github.com/zaps166/QMPlay2/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/QMPlay2-${PV}"
+fi
+
+LICENSE="LGPL-3"
+SLOT="0"
+
+IUSE="avdevice +audiofilters avresample alsa cdio cuvid dbus +extensions
+	gme inputs +lastfm libass lyrics mediabrowser modplug mpris2
+	notifications opengl portaudio pulseaudio sid svg taglib vaapi
+	vdpau +videofilters visualizations xv"
+
+REQUIRED_USE="
+	audiofilters? ( || ( alsa portaudio pulseaudio ) )
```

Comment:
You `+audiofilters` yet none of these, so the emerge stops right away asking user to enable some of these. You should enable something by default, like `+alsa`. 

... which I did and got
```
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find ALSA (missing: ALSA_LIBRARY ALSA_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake/Modules/FindALSA.cmake:60 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  src/modules/ALSA/CMakeLists.txt:20 (find_package)


-- Configuring incomplete, errors occurred!
See also "/tmp/portage/media-video/qmplay2-19.12.19/work/qmplay2-19.12.19_build/CMakeFiles/CMakeOutput.log".
 * ERROR: media-video/qmplay2-19.12.19::testworld failed (configure phase):
 *   cmake failed
```